src/content/developers/docs/oracles/index.md

```diff
@@ -1,314 +1,90 @@
 ---
 title: Oracles
-description: Oracles help get real-world data into your Ethereum application because smart contracts can't query real-world data on their own.
+description: Oracles provide Ethereum smart contracts with access to real-world data, unlocking more use-cases and greater value for users.  
 lang: en
 sidebar: true
 incomplete: true
 ---
 
-Oracles are data feeds that connect Ethereum to off-chain, real-world information so that you can query data in your smart contracts. For example, prediction market dapps use oracles to settle payments based on events. A prediction market may ask you to bet your ETH on who will become the next president of the United States. They'll use an oracle to confirm the outcome and payout to the winners.
+Oracles extract information from the real world and put it on the blockchain for smart contracts to use. This is necessary because smart contracts running on Ethereum cannot access information stored outside the blockchain network. 
```

Comment:
```suggestion
Oracles are data feeds that bring data from off the blockchain (off-chain) data sources and puts it on the blockchain (on-chain) for smart contracts to use. This is necessary because smart contracts running on Ethereum cannot access information stored outside the blockchain network. 
```

I don't like the use of real-world much if I'm honest. This is all real world, I dont think any data feed is more real world than anything on Ethereum.